src/sequelize.js

```diff
@@ -778,9 +778,10 @@ Use Sequelize#query if you wish to use replacements.`);
    * {@link Model.schema}
    *
    * @param {string} schema Name of the schema
-   * @param {object} [options={}] query options
-   * @param {boolean|Function} [options.logging] A function that logs sql queries, or false for no logging
-   *
+   * @param {object} [options={}] CreateSchemaQueryOptions
+   * @param {string} [options.collate=null]
+   * @param {string} [options.charset=null]
+    *
    * @returns {Promise}
    */
   async createSchema(schema, options) {
```

Comment:
Maybe we should move this to `sequelize-typescript.ts`? If we want to keep supporting this after QueryInterface is public. cc @ephys

Reply:
Will implement and resolve after feedback from @ephys

Reply:
Though  big gripe I've seen from the community is too many ways to do things - may be worth suppressing this method and keeping it in QueryInterface

Reply:
I agree, creating schemas isn't something we do every 5 minutes, there is no reason to have a shortcut method.

Plus, migrations already receive queryInterface instead of the sequelize instance

Model also had a createSchema method for some reason, which I think we removed in v7

I'm pro deprecating this method (doesn't have to be in this PR)

I'm also pro-deprecating `getQueryInterface`, it's redundant with `queryInterface`